create.html

```diff
@@ -1560,7 +1560,16 @@ <h4 class="modal-body text-danger" :style="{display: !getID() ? 'block' : 'none'
         document.getElementById('preview').contentWindow.addEventListener("error", function(event) {
           var error = event.error
           if (error.type == "ImageLoadError") {
-            app.error = {message: "There is a problem with your image URL: " + error.url, lineno: code.substring(0, code.indexOf(error.url)).split('\n').length}
+              if (error.url === "") {
+                function findErrorType(type){
+                return (code.indexOf(type) != -1)
```

Comment:
your formatting here is off
```suggestion
                  return (code.indexOf(type) != -1)
```